### DISCUSSION TITLE
国产化arm版本集群milvus-v2.4.6部署

### DISCUSSION THREAD

**Discussion by jhyever** [milvus-io/milvus#38611] [Announcement]
部署x86集群时是这些镜像，部署arm版本镜像这戏都需要拉取arm版镜像么， 避免漏镜像，有直接拿到国产化部署arm镜像需要的资源包么
<img width="200" alt="ac49ba23f6dec3bda8b735fc0b537c8" src="https://github.com/user-attachments/assets/6d83b22f-3be8-42e2-86e7-4938f396fcfb" />

**Comment by yhmo** [comment]
@JackLCL could help.

**Reply by jhyever** [reply]
麻烦帮忙回复一下这个关于：国产化arm版本集群部署milvus的问题，感谢

**Reply by JackLCL** [reply]
已在微信沟通
